docs/devops-guide/docker.md

```diff
@@ -6,15 +6,18 @@ sidebar_label: Docker
 
 ## Quick start
 
-In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
-follow these steps:
+In order to quickly run Jitsi Meet on a machine running Docker, make use of Docker Compose and follow these steps:
 
 * Download and extract the [latest release]
+  * `wget $(curl -s https://api.github.com/repos/jitsi/docker-jitsi-meet/releases/latest | grep 'zip' | cut -d\" -f4)` (change zip to tar.gz if you prefer)
+  * `unzip <filename>`
   * Alternatively, to test the latest changes clone the repository: `git clone https://github.com/jitsi/docker-jitsi-meet && cd docker-jitsi-meet`
 * Create a ``.env`` file by copying and adjusting ``env.example``
   * `cp env.example .env`
 * Set strong passwords in the security section options of ``.env`` file by running the following bash script
   * `./gen-passwords.sh`
+* Adjust configuration options
```

Comment:
This doubles a line above: 
> and adjusting ``env.example``

Nonetheless I agree it should be changed, maybe you could change it to:
```
Copy the env.example file with:
cp env.example .env
Then edit and adjust the configuration options, for example with nano or another editor:
nano .env
```

Reply:
Yes, you're right. Good suggestion. Just to be sure: adjustments can be made _before_ running gen-passwords.sh - i.e. they don't get overwritten?

Reply:
Good question, I looked at:
https://github.com/jitsi/docker-jitsi-meet/blob/master/gen-passwords.sh
And it seems it just replaces some lines with new passwords, so these are the only lines that get changed.